### DISCUSSION TITLE
how to change port 8080 of filebrowser??

### DISCUSSION THREAD

**Discussion by techsavvybyte** [tteck/Proxmox#1501] [Help Needed] [answered]
Hi I need to change port 8080 of file browser in proxmox. how I can do this. I didnt find anythung to change port please help

**Comment by tteck** [comment, marked as answer]
Change the port (8081) and password (changeme) to what you want. Then, execute the snippet in the LXC console, Reboot.

```
rm filebrowser.db
filebrowser config init -a '0.0.0.0'
filebrowser config set -a '0.0.0.0'
filebrowser users add admin changeme --perm.admin
filebrowser config init -p '8081'
filebrowser config set -p '8081'
```

**Reply by techsavvybyte** [reply]
I have tried but its say filebrowser.db already exist

**Reply by tteck** [reply]
`rm filebrowser.db`!
